### PR TITLE
Issue 22

### PR DESCRIPTION
**Pull request by rameezjhaveri**
- [x] Merge is going to correct branch.
- [x] Code is well formated (readable)
- [x] Code uses sensible variable/method names (readable)
- [x] Peer Reviewed
- [x] Reviewed by Tadas

### CODE REVIEW

File: app/views/header.scala.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
                 <a class="nav-link" href="/gallery">@messages("header.gallery")</a>
             </li>
             <li class="nav-item">
-            @if(loggedIn) {
+            @if(user.length>2) {
```
Comment (tvaidotas) [inline app/views/header.scala.html]: `user.length>2` I don't understand the logic here

File: app/views/header.scala.html [hunk 2]
```diff
@@ -25,14 +25,14 @@
                 <a class="nav-link" href="/contact">Contact</a>
             </li>
 <!-- TO BE SHOWN IF LOGGED IN -->
-            @if(loggedIn==true) {
+            @if(user.length>2) {
```
Comment (tvaidotas) [inline app/views/header.scala.html]: same

File: app/views/header.scala.html [hunk 3]
```diff
@@ -25,14 +25,14 @@
...
                 <li class="nav-item dropdown">
                     <a class="nav-link dropdown-toggle" id="navbarDropdownMenuLink-4" data-toggle="dropdown"
                     aria-haspopup="true" aria-expanded="false">
                         <i class="fas fa-user"></i> Profile </a>
                     <div class="dropdown-menu dropdown-menu-right dropdown-info" aria-labelledby="navbarDropdownMenuLink-4">
-                        <a class="dropdown-item" href="#">My account</a>
-                        <a class="dropdown-item" href="#">Log out</a>
+                        <a class="dropdown-item" href="/account">My account</a>
```
Comment (tvaidotas) [inline app/views/header.scala.html]: `My account` should come from messages and have welsh translation

File: app/views/header.scala.html [hunk 4]
```diff
@@ -25,14 +25,14 @@
...
+                        <a class="dropdown-item" href="/logout">Log out</a>
```
Comment (tvaidotas) [inline app/views/header.scala.html]: `Log out` should come from messages and have welsh translation

File: app/views/login.scala.html [hunk 5]
```diff
@@ -10,6 +10,7 @@ <h3>Please log in</h3>
         @helper.inputPassword(form("password"), '_label -> "Password", '_showConstraints -> false)
         <button type="submit">Log In</button>
         <a href="/sign-up"><button type="button">Sign Up</button></a>
+        <a href="/"><button type="button">Cancel</button></a>
```
Comment (tvaidotas) [inline app/views/login.scala.html]: `Cancel` should come from messages and have welsh translation

**Review by tvaidotas — CHANGES_REQUESTED** [review]

File: app/views/header.scala.html (hunk 1 above)
Comment (rameezjhaveri) [reply app/views/header.scala.html]: when creating an account, the minimum length of a valid username is 3 chars so >2 accepts valid users

Comment (rameezjhaveri) [reply app/views/header.scala.html]: Also allows me to pass user as a string for both validations and to display user name

**Review by tvaidotas — APPROVED** [review]

**Pull request merged**